lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -87,10 +124,15 @@ class I18nRegressionTests(BaseI18nTestCase):
     """
     Tests for i18n
     """
+
+    def setUp(self):
+        super(I18nRegressionTests, self).setUp()
+        self.user_creation_login()
+
     def test_es419_acceptance(self):
         # Regression test; LOC-72, and an issue with Django
         self.release_languages('es-419')
-        response = self.client.get('/', HTTP_ACCEPT_LANGUAGE='es-419')
+        response = self.client.get('/courses', HTTP_ACCEPT_LANGUAGE='es-419')
```

Comment:
@cahrens This has been changed back to '/' where possible. 

When the user is logged in the '/' path will not return the html and the required tag used here in testing.  This was true in other tests that required a logged in user prior to my changes.

I have changed the tests to only logging in a user when it is needed and reverted the tests that do not require a logged in user to '/'.